Add excludes directories

I use the Visual Studio Code(vscode) with local history plugin, which generates a .history folder to hold the old content, so when I run the swag init command, it checks the folder and reports an error. 

```
2018/05/22 09:40:16 Generate swagger docs....
2018/05/22 09:40:16 Generate general API Info
2018/05/22 09:40:16 ParseFile panic:.history\main_20180423195313.go:1:1: expected 'package', found 'EOF'
panic: ParseFile panic:.history\main_20180423195313.go:1:1: expected 'package', found 'EOF'
```

So I want cli to have a parameter that excludes some directories.

--- a/parser.go
+++ b/parser.go
@@ -660,6 +660,8 @@ func (parser *Parser) getAllGoFileInfo(searchDir string) {
 		//exclude vendor folder
 		if ext := filepath.Ext(path); ext == ".go" &&
 			!strings.Contains(string(os.PathSeparator)+path, string(os.PathSeparator)+"vendor"+string(os.PathSeparator)) &&
+			!strings.Contains(string(os.PathSeparator)+path, string(os.PathSeparator)+".history"+string(os.PathSeparator)) &&
+			!strings.Contains(string(os.PathSeparator)+path, string(os.PathSeparator)+".idea"+string(os.PathSeparator)) &&
 			!strings.Contains(string(os.PathSeparator)+path, string(os.PathSeparator)+".git"+string(os.PathSeparator)) {
 			fset := token.NewFileSet() // positions are relative to fset
 			astFile, err := goparser.ParseFile(fset, path, nil, goparser.ParseComments)

